app/routes/chatbot.js

```diff
@@ -24,143 +20,17 @@ const mapRequestParamsMiddleware = require('../../lib/middleware/map-request-par
 const lookUpUserMiddleware = require('../../lib/middleware/user-get');
 const createNewUserIfNotFoundMiddleware = require('../../lib/middleware/user-create');
 const processBroadcastConversationMiddleware = require('../../lib/middleware/broadcast');
+const getPhoenixCampaignMiddleware = require('../../lib/middleware/campaign-get');
```

Comment:
Similar thought, would it make sense to specifically spell this out as `phoenix-campaign-get` in the filename since getting a Campaign could potentially refer to querying Contentful.